### DISCUSSION TITLE
How do yo use $t() inside Alpine.store?

### DISCUSSION THREAD

**Discussion by simonferndriger** [rehhouari/alpinejs-i18n#4] [General]
`Alpine.store('categories', [ this.$t('all') ], ...)` does't work ($t seems to be undefined)?

**Comment by rehhouari** [comment]
`AlpineI18n.t()` could be used from javascript so it would work inside `Alpine.store` :)
see: https://github.com/rehhouari/alpinejs-i18n#usage-from-javascript

**Reply by simonferndriger** [reply]
@rehhouari Thanks :)
